src/pycram/description.py

```diff
@@ -360,16 +425,24 @@ def get_origin_transform(self) -> Transform:
     @property
     def color(self) -> Color:
         """
-        The getter for the rgba_color of this link.
+        The rgba_color of this link.
 
         :return: A Color object containing the rgba_color of this link.
         """
         return self.world.get_link_color(self)
 
+    def set_color(self, color: Color) -> None:
+        """
+        Set the color of this link, could be rgb or rgba.
+
+        :param color: The color as a list of floats, either rgb or rgba.
+        """
+        self.color = color
+
     @color.setter
     def color(self, color: Color) -> None:
         """
-        The setter for the color of this link, could be rgb or rgba.
```

Comment:
This kinda bites with the thing above. Is is a property? Should i use getters and setters?

Reply:
yes it is, added a deprecation warning in the color setter.